doc/guides/abstraction/posix.rst

```diff
@@ -0,0 +1,434 @@
+
+
+POSIX Support
+#############
+
+The Portable Operating System Interface (POSIX) is a family of standards
+specified by the IEEE Computer Society for maintaining compatibility between
+operating systems. Zephyr implements a subset of the embedded profiles PSE51
+and PSE52 and BSD Sockets.
+
+With the POSIX support available in Zephyr, an existing POSIX threading
```

Comment:
I'd removed word "threading", we're already much wider-scope than just Pthreads.